app/src/main/java/org/jellyfin/androidtv/ui/FavoriteBadge.kt

```diff
@@ -0,0 +1,34 @@
+package org.jellyfin.androidtv.ui
+
+import android.content.Context
+import android.view.View
+import android.widget.ImageView
+import org.jellyfin.androidtv.R
+import org.jellyfin.androidtv.model.itemtypes.BaseItem
+
+class FavoriteBadge(context: Context, val item: BaseItem? = null) : ImageView(context) {
+	private val changeListener: () -> Unit = {
+		item?.let(::setItemData)
+	}
+
+	init {
+		setImageDrawable(context.getDrawable(R.drawable.ic_heart_red))
+		item?.let(::setItemData)
+	}
+
+	constructor(context: Context) : this(context, null)
```

Comment:
This constructor shouldn't be necessary because  the `item` parameter has a default value